modules/installation-creating-gcp-dns.adoc

```diff
@@ -33,9 +37,18 @@ requires.
 
 . Export the following variable required by the resource definition:
 +
+ifndef::shared-vpc[]
 ----
 $ export CLUSTER_NETWORK=`gcloud compute networks describe ${INFRA_ID}-network --format json | jq -r .selfLink`
```

Comment:
The network is hosted by host project and current active service account works for service project, so we need to add --project --account to specify host project and service account of host project. For example:
export CLUSTER_NETWORK=$(gcloud compute networks describe ${HOST_PROJECT_NETWORK} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)

Reply:
@kalexand-rh Please add --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} to the cli:
$ export CLUSTER_NETWORK=`gcloud compute networks describe ${INFRA_ID}-network --format json | jq -r .selfLink`

Reply:
@kalexand-rh Please be aware of this. This issue has not been addressed yet.